packages/transport/src/modules/accounts/AccountController.ts

```diff
@@ -235,11 +236,29 @@ export class AccountController {
     public async syncDatawallet(force = false): Promise<void> {
         if (!force && !this.autoSync) return;
 
-        await this.synchronization.sync("OnlyDatawallet");
+        const changedItems = await this.synchronization.sync("OnlyDatawallet");
+        await this.triggerEventsForChangedItems(changedItems);
     }
 
     public async syncEverything(): Promise<ChangedItems> {
-        return await this.synchronization.sync("Everything");
+        const changedItems = await this.synchronization.sync("Everything");
+        await this.triggerEventsForChangedItems(changedItems);
+
+        return changedItems;
+    }
+
+    private async triggerEventsForChangedItems(changedItems: ChangedItems) {
+        const changedIdentityDeletionProcessIds = changedItems.changedObjectIdentifiersDuringDatawalletSync.filter((x) => x.startsWith("IDP"));
```

Comment:
I'd prefer `const idsOfChangedIdentityDeletionProcesses` here.

Reply:
hmm. me not.